thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -84,6 +41,7 @@ export default Ember.Route.extend(AuthenticatedRouteMixin, {
 
     if (anomalyUrn) {
       anomalyContext = fetch(`/rootcause/raw?framework=anomalyContext&urns=${anomalyUrn}`).then(checkStatus).catch(res => undefined);
+      anomalySessions = fetch(`/session/query?anomalyId=${anomalyId}`).then(checkStatus).catch(res => undefined);
```

Comment:
This will save a promise into `anomalySessions` is that what we want?

Reply:
yes, they all become part of the RSVP hash